contracts/ERC721A.sol

```diff
@@ -428,7 +411,7 @@ contract ERC721A is Context, ERC165, IERC721, IERC721Metadata, IERC721Enumerable
      * @param _data bytes optional data to send along with the call
      * @return bool whether the call correctly returned the expected magic value
      */
-    function _checkOnERC721Received(
+    function _checkOnERC721AReceived(
```

Comment:
This function name could be better as staying generic since the -A version does not change anything over baseline ERC-721.

Reply:
I agree, but I had to change it because openzeppelin's ERC721Upgradeable has it as a non-overrideable private function and I needed to be able to call it in ERC721A